Source/JavaScriptCore/heap/MarkedSpace.cpp

```diff
@@ -358,6 +358,20 @@ bool MarkedSpace::isPagedOut()
     return pagedOutPagesStats.mean() > pagedOutPagesStats.count() * bailoutPercentage;
 }
 
+MarkedBlock::Handle* MarkedSpace::findMarkedBlockHandle(MarkedBlock* block)
```

Comment:
let's not forget to remove these findMarkedBlockHandle methods as well once we're done with this debugging code

Reply:
Let's rename this to `findMarkedBlockHandleDebug`.